crates/sui-graphql-rpc/schema/draft_schema.graphql

```diff
@@ -421,15 +421,22 @@ type Epoch {
 
 type ProtocolConfigs {
   protocolVersion: Int!
-  configs: [ProtocolConfig]
-  config(key: String!): ProtocolConfig
+  featureFlags: [ProtocolConfigFeatureFlag]
+  configs: [ProtocolConfigAttr]
+  config(key: String!): ProtocolConfigAttr
+  featureFlag(key: String!): ProtocolConfigFeatureFlag
 }
 
-type ProtocolConfig {
+type ProtocolConfigAttr {
```

Comment:
curious what are the application usecases of reading ProtocolConfigAttr and ProtocolConfigFeatureFlag?

Reply:
Feature flags are boolean only, and they are stored in their own structure within `ProtocolConfig` which means that feature flag names can shadow other attribute names.